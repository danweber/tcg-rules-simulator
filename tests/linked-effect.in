VERSION:2

# most basic link test

# get +760 from DW1-01, +1500 DP boost from plugged card
FIELD1:Linked,DW1-Van,CS3-02,PLUG,DW1-01,CS3-02,CS3-02
GAME:P1 T3 MAIN 5
DUMP:DP,KEYWORDS

EXIT
