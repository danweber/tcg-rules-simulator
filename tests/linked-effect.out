FIELD1:Linked,DW1-Vanillamon,CS3-Fairyfeatherlink,PLUG,DW1-Bigmon,CS3-Fairyfeatherlink,CS3-Fairyfeatherlink,8000DP,DODGE
FIELD2:
GAME:P1 T3 MAIN 5
